routes/images.js

```diff
@@ -2,21 +2,22 @@ const express = require('express');
 const router = express.Router();
 const axios = require('axios');
 
-router.get('/getProducts', async (req, res) => {
+router.get('/getImages', async (req, res) => {
   try {
-    const { category, gender, userId } = req.query;
-
-    // Validate and set default parameters
+    const { q, limit, minPrice, maxPrice, dept, sort, offset } = req.query;
```

Comment:
The API should only accept category, user and gender. It cannot provide the other parameters.